src/http.cc

```diff
@@ -119,6 +119,7 @@ int HttpPost(const std::string &url, const std::string &data,
   // HTTP.
   curl_easy_setopt(curl, CURLOPT_FOLLOWLOCATION, true);
   curl_easy_setopt(curl, CURLOPT_USERAGENT, kUserAgent);
+  curl_easy_setopt(curl, CURLOPT_PROXY, proxy.c_str());
```

Comment:
This is a change in behavior from the current code, because apparently:

'Setting the proxy string to "" (an empty string) will explicitly disable the use of a proxy, even if there is an environment variable set for it.'

Can you wrap this in `if (!proxy.empty())`?

Reply:
Indeed, I will add the wrapper.